src/providers/twitch/TwitchAccount.cpp

```diff
@@ -264,6 +264,25 @@ void TwitchAccount::loadUserstateEmotes(std::weak_ptr<Channel> weakChannel)
             [this, weakChannel](QJsonArray emoteSetArray) {
                 auto emoteData = this->emotes_.access();
                 auto localEmoteData = this->localEmotes_.access();
+
+                std::set<QString> subscriberChannelIDs;
+                for (auto emoteSet : emoteSetArray)
+                {
+                    IvrEmoteSet ivrEmoteSet(emoteSet.toObject());
+                    if (!ivrEmoteSet.tier.isNull())
+                    {
+                        subscriberChannelIDs.insert(ivrEmoteSet.channelId);
+                    }
+                }
+
+                for (auto emoteSet : emoteData->emoteSets)
```

Comment:
As clang-tidy said, this should be a const reference.

Reply:
I wasn't sure if that one was good, usually const suggestions are poo but I did notice that one referenced a different rule

Reply:
It's a bit confusing. Clang-tidy warns when local variables aren't declared `const` which we don't want (I'm pretty sure there was a PR that fixed this - [`misc-const-correctness.AnalyzeValues`](https://clang.llvm.org/extra/clang-tidy/checks/misc/const-correctness.html#cmdoption-arg-AnalyzeValues)).